app/serializers/folder_serializer.rb

```diff
@@ -21,6 +21,7 @@ class FolderSerializer < ActiveModel::Serializer
   attributes :id, :name, :team_id, :team_name, :description
 
   belongs_to :team
+  has_many :accounts, serializer: AccountMinimalSerializer
```

Comment:
This might break things? Do we never access anything in accounts through folders that is not in the AccountMinimalSerializer?